integration-tests/tests/rpc/creditcoin/getEvents.spec.ts

```diff
@@ -0,0 +1,46 @@
+// [object Object]
+// SPDX-License-Identifier: Apache-2.0
+
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import type { Event, Vec } from '@polkadot/types-codec';
+
+describe('Creditcoin RPC', (): void => {
+  let api: ApiPromise;
+
+  beforeEach(async () => {
+    process.env.NODE_ENV = 'test';
+
+    const provider = new WsProvider('ws://127.0.0.1:9944');
+
+    api = await ApiPromise.create({
+      provider: provider,
+      rpc: {
+        creditcoin: {
+          getEvents: {
+
+            params: [
+              {
+                name: 'at',
+                type: 'Hash',
+                isOptional: true
+              }
+            ],
+            type: 'Vec<Event>'
+          }
+        }
+      }
+    });
+  });
+
+  afterEach(async () => {
+    await api.disconnect();
+  });
+
+  it('getEvents() should return some events', (): void => {
+    return api.rpc.creditcoin.getEvents().then(result => {
+console.log(`**** DEBUG: result=${result}`);
```

Comment:
@nathanwhit with the return type defined as `Vec<Event>` above the actual data returned is 
```bash
    **** DEBUG: result=[{"index":"0x0000","data":null}, {"index":"0x0000","data":null}]
```

while when using curl I see 

```json
[{"ctcDeposit":{"amount":"28000000000000000000","into":"5DkPYq8hFiCeGxFBkz6DAwnTrvKevAJfTYrzFtr9hpDsEAU1"}},{"rewardIssued":{"amount":"28000000000000000000","to":"5DkPYq8hFiCeGxFBkz6DAwnTrvKevAJfTYrzFtr9hpDsEAU1"}}]
```

I am not able to find the type definitions for `ctcDeposit` or `friendly::Event` anywhere in `src/interfaces/`.  Do I have to build this myself when constructing the API or should they be available for import somehow ?